docs/source/customize/handler.rst

```diff
@@ -61,3 +61,38 @@ Then we can start using the handler:
         handler: the_handler_for_my_server
 
 That's all you need to do
+
+------
+
+As a alternative but not recommended way, you may provide your handler by a single ``.py`` file, rather than a plugin
+
+Put the same code as above, without the ``on_load`` method, into a ``.py`` file, ``my_handler.py`` for example, then use it as follows:
+
+1.  Place it into a valid python package in the working directory of MCDR, e.g.:
+
+    .. code-block:: diff
+
+            my_mcdr_server/
+        ++  ├─ handlers/
+        ++  │   ├─ __init__.py
+        ++  │   └─ my_handler.py
+            │
+            ├─ server/
+            ├─ config.yml
+            └─ permission.yml
+
+    This make your handler class accessible with the following python code:
```

Comment:
This might be misleading, cuz the user might incorrectly try adding the following code to unexpected places

You can replace the code block with an inline block to make it look less like a guide

```rst
This make your handler class accessible with ````
```

Reply:
actually that was the same as the old documentation ;)